source/PlayState.hx

```diff
@@ -9,8 +9,30 @@ class PlayState extends FlxState
 	{
 		super.create();
 
-		var myHero = new Hero(300, 300);
-		add(myHero);
+		var text = new flixel.text.FlxText(0, 0, 0, "Hello World!", 24);
+		text.borderColor = 0xFFFF0000;
+		text.borderSize = 2.0;
+		text.borderStyle = SHADOW;
+		text.screenCenter();
+		add(text);
+
+		var hero1 = new ScreensaverHero();
+		var hero2 = new ScreensaverHero();
+		var hero3 = new ScreensaverHero();
+		var hero4 = new ScreensaverHero();
+
+		hero1.screenCenter();
+		hero2.screenCenter();
+		hero3.screenCenter();
+		hero4.screenCenter();
+
+		add(hero1);
+		add(hero2);
+		add(hero3);
+		add(hero4);
```

Comment:
Loops are your friend here, something like:
```
for (i in 0...4) {
    var hero = new ScreensaverHero();
    hero.screenCenter();
    add(hero);
}
```

This gets the job done, which is a fine result when you're under time pressure, but it's not hard to see the pain that repetition brings. That, and it doesn't hurt to get more comfortable with the fundamentals of the Haxe language.

No action required here, but optionally swap to a loop if you want the practice.